CLR/Graphics/Graphics.cpp

```diff
@@ -195,7 +195,7 @@ HRESULT CLR_GFX_Bitmap::CreateInstance( CLR_RT_HeapBlock& ref, const CLR_UINT8*
         /* When loading a Windows BMP, GIF, or JPEG file, it is converted in-place to the native BPP.
          * When loading a compressed TinyCLR Bitmap from a resource file, two bitmaps are needed to decompress, then convert.
          * This fragments the heap and wastes space until the next garbage collection is done.
-         * When using the SimpleHeap, there is no relocation, so decompressing into a temp bitmap into the simpleheap wastes
+         * When using the heap, there is no relocation, so decompressing into a temp bitmap into the heap wastes

```

Comment:
The comment shouldn't just blindly replace words with a GREP, it needs to be re-worded to reflect the consequences. The platform creator should still have the option of creating a simple heap/big object storage to support large bitmaps that are otherwise not supported in the managed heap. (One of the implementation details in the managed heap is an assumption on the maximum size of any single allocation)